package/MDAnalysis/coordinates/memory.py

```diff
@@ -501,9 +501,14 @@ def timeseries(self, asel=None, start=0, stop=-1, step=1, order='afc'):
             of the underlying numpy array is returned, while a copy of the
             data is returned whenever `asel` is different from ``None``.
         start : int (optional)
+            the start trajectory frame
         stop : int (optional)
+            the end trajectory frame
+            .. deprecated:: 2.4.0
+            Note that `stop` is currently *inclusive* but will be
+            changed in favour of being *exclusive* in version 3.0.  
```

Comment:
```suggestion

            .. deprecated:: 2.4.0
               Note that `stop` is currently *inclusive* but will be
               changed in favour of being *exclusive* in version 3.0.  

```
Reformatting broke docs, see: https://mdanalysis--3894.org.readthedocs.build/en/3894/documentation_pages/coordinates/memory.html

Reply:
Yes, you need an empty line above reST paragraph constructs.